File: rb-appscript/tags/rb-appscript-0.5.3/sample/Select_all_HTML_files.rb
#!/usr/bin/env ruby

# Selects all .htm/.html files in the top Finder window.

# Note: if using the appscript gem, rubygems must be required first:
begin; require 'rubygems'; rescue LoadError; end

require "appscript"
include Appscript

finder = app('Finder')
finder.activate
folder = finder.Finder_windows[1].target.get
folder.files[its.name_extension.is_in(['htm', 'html'])].select

File: py-appscript/trunk/appscript_3x/sample/appscript/Select_all_HTML_files.py
#!/usr/bin/env python3

# Selects all .htm/.html files in the top Finder window.

from appscript import *

finder = app('Finder')

finder.activate()

w = finder.Finder_windows[1].target.get()
w.files[its.name_extension.isin(['htm', 'html'])].select()